src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -513,4 +513,28 @@
   <data name="PX1007DacProperty" xml:space="preserve">
     <value>DAC property</value>
   </data>
+  <data name="PX1019Title" xml:space="preserve">
+    <value>The AutoNumber attribute can be placed only on DAC property with string type</value>
+  </data>
+  <data name="PX1020Title" xml:space="preserve">
+    <value>The length of a string must be greater than {0} for a DAC property with an AutoNumber attribute</value>
+  </data>
+  <data name="PX1019Fix" xml:space="preserve">
+    <value>Change DAC property type to string</value>
+  </data>
+  <data name="PX1020Fix" xml:space="preserve">
+    <value>Make string length sufficient for autonumbering</value>
+  </data>
+  <data name="PX1022DacFix" xml:space="preserve">
```

Comment:
@k-s-popova please review PX1022 diagnostic messages

Reply:
@SENya1990 , please change as follows:
PX1022DacTitle: Non-public DAC extensions are not supported
PX1022GraphTitle: Non-public graph extensions are not supported
The fixes are OK.

Reply:
Fixed